lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -6,20 +6,22 @@ module Exceptions
     # you must define the minor code in the locales file and call this class from
     # raise_error middleware.
     class BackendServiceException < BaseError
-      attr_reader :response_values, :original_status, :original_body
+      attr_reader :service_response
 
-      def initialize(key = nil, response_values = {}, original_status = nil, original_body = nil)
-        @response_values = response_values
+      def initialize(key = nil, service_response = {})
         @key = key || 'VA900'
-        @original_status = original_status
-        @original_body = original_body
+        @service_response = service_response
         validate_arguments!
       end
 
+      def trigger_breakers?
+        (500..599).cover?(original_status || status)
+      end
+
       # The message will be the actual backend service response from middleware,
       # not the I18n version.
       def message
-        "BackendServiceException: #{response_values.merge(code: code)}"
+        "BackendServiceException: #{code} - #{detail || service_response[:detail]}"
```

Comment:
looks like detail is already doing this, and the comment is no longer applicable. I'll change this.